keyboards/work_louder/work_board/info.json

```diff
@@ -4,120 +4,202 @@
     "url": "https://worklouder.cc/",
     "maintainer": "Work Louder",
     "usb": {
+      "max_power": 100,
       "vid": "0x574C",
       "pid": "0xDCD1"
     },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["D3", "D5", "D4", "D6", "D7", "B4", "B5", "B6", "C6", "C7", "F7", "F6", "E6"],
+        "rows": ["F0", "F1", "F4", "F5"]
+    },
     "encoder": {
       "rotary": [
         {"pin_a": "B0", "pin_b": "B1"}
       ]
     },
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "rgblight": true
+    },
+    "build": {
+        "lto": true
+    },
     "processor": "atmega32u4",
     "bootloader": "atmel-dfu",
+    "rgb_matrix": {
+        "driver": "WS2812",
+        "layout": [
+            { "flags": 1, "matrix": [3, 8], "x": 223, "y": 63 },
```

Comment:
Probably should hold off on this, there are some issues with the way `qmk info` parses the `g_led_config`.

Reply:
Why would putting this in info.json have anything to do with the parsing logic?

Reply:
https://github.com/qmk/qmk_firmware/pull/19954

Reply:
I can tell this was copied from `qmk info` output due to the key order and spacing around the braces.

Reply:
Ah, yeah, it's off.  Fixed manually.

Reply:
![image](https://user-images.githubusercontent.com/4781841/222031362-3098af67-b7ad-40d9-b340-8f2d9beb1cc4.png)
![image](https://user-images.githubusercontent.com/4781841/222031380-9849fc75-9aa9-456b-add7-ea5d969a8ee2.png)
LGTM